tests/kernel/kernel.c

```diff
@@ -59,8 +59,16 @@ int call_test(const char *name)
     }
   CHECK_OPENCL_ERROR_IN ("clCreateKernelsInProgram call failed");
 
+  char kernel_name[1024];
   for (cl_uint i = 0; i < num_kernels; i++)
     {
+      size_t name_len = 0;
+      err = clGetKernelInfo (kernels[i], CL_KERNEL_FUNCTION_NAME, 1024,
+                             kernel_name, &name_len);
+      CHECK_OPENCL_ERROR_IN ("clGetKernelInfo failed");
+      kernel_name[name_len] = 0;
+      if (name_len && strstr (kernel_name, "DONT_RUN"))
```

Comment:
Not sure of this commit. Why not use volatile instead?

Reply:
fixed